.github/workflows/build.yml

```diff
@@ -123,3 +123,8 @@ jobs:
           # this should be a last resort in case by any chances the build memory gets too high
           # but in general this should never happen
           NODE_OPTIONS: '--max_old_space_size=4096'
+
+      - name: Sync Orama Cloud
+        if: github.ref == 'refs/heads/main'
+        run: |
+          npm run sync-orama
```

Comment:
@ovflowd I need your support here to export two environment variables that are required for the script to work: `ORAMA_INDEX_ID` and `ORAMA_SECRET_KEY`. If you could give me the token names you used on GitHub, I can insert them here

Reply:
The secret names are the same as you gave me on Slack.

Reply:
![image](https://github.com/nodejs/nodejs.org/assets/12037269/e7b71b08-31cb-4d94-9989-492be52a3a02)

But these secrets will not work on PRs coming from forks.